doc/source/_static/custom.css

```diff
@@ -1,17 +1,49 @@
-/*This ensures that clickable links in the SG examples are the right color*/
-div.section a span {
-    color: #2980B9 !important
+/* colors */
+
+h1 {
+    color: #139C5A;
+}
+
+h2 {
+    color: #333333;
 }
 
-/*Copied from sphinx' basic.css to ensure the sphinx >2.0 docstrings are
-rendered somewhat properly (xref https://github.com/numpy/numpydoc/issues/215) */
+.nav li.active>a, .navbar-nav>.active>.nav-link {
+    color: #139C5A!important;
+}
+
+.toc-entry>.nav-link.active {
+    border-left-color: #139C5A;
+    color: #139C5A!important;
+}
+
+.nav li>a:hover {
+    color: #333333!important;
+}
+
+/* buttons */
+
+.button>p>a {

```

Comment:
Did you copy this from somewhere? (in which case maybe worth adding a comment about it)

We could maybe also use https://sphinx-panels.readthedocs.io/en/latest/ for this (but that can be done later as a follow-up issue)

Reply:
Nope, buttons are brand new :). I'll keep in mind the panel option.